### PR TITLE
addresses Neil's second draft comments

### PR DESCRIPTION
**Pull request by marcfishelquantil**

### CODE REVIEW

File: docs/portal/dashboard.md [hunk 1]
```diff
@@ -12,28 +12,28 @@ Resource quota charts at the top of the Dashboard show “quota consumed” and
...
+<p align=center><img src="/docs/resources/images/dashboard/dashboard-metrics-cpu.png" width="600">
 
-![null](</docs/resources/images/dashboard/dashboard-metrics-traffic.png>)
+<p align=center><img src="/docs/resources/images/dashboard/dashboard-metrics-memory.png" width="600">
 
-![null](</docs/resources/images/dashboard/dashboard-metrics-bandwidth.png>)
+<p align=center><img src="/docs/resources/images/dashboard/dashboard-metrics-traffic.png" width="600">
 
-![null](</docs/resources/images/dashboard/dashboard-metrics-storage.png>)
+<p align=center><img src="/docs/resources/images/dashboard/dashboard-metrics-bandwifth.png" width="600">
```
Comment (Jianghai-Quantil) [inline docs/portal/dashboard.md:26]: The link to image is broken. There is a typo in the image name. "bandwifth" shall be changed to "bandwidth".

File: docs/portal/image-registry/managing-repositories-under-a-project.md [hunk 2]
```diff
@@ -5,8 +5,8 @@ A project can have one or more repositories and a repository can have one or mor
...
-![null](</docs/resources/images/registry/repository-description.png>)
-4. To delete a repository, click its **Actions** menu, and then select **Delete**. When prompted, click **Yes, delete!**
\ No newline at end of file
+2. On the Registry Projects page, click the name of a project. The detail page appears. This page has two sections:<ul><li>**Basic information**  shows read-only information about the selected project.</ul></li><ul><li>**Repositories** shows the repositories associated with the selected project. A field at the top left allows you to filter the repositories shown.</ul></li>
+<p align=center><img src="/docs/resources/images/registry/edit-repository.png" width="600"></p>
+3. To change a repository's description, click the pencil icon in the <strong>Description</strong> column. When the field opens, change the description, and then click the check mark.
+<p align=center><img src="/docs/resources/images/registry/repository-description.png" width="600"></p>
+4. To delete a repository, click its <strong>Actions</strong> menu, and then select **Delete**. When prompted, click <strong>Yes, delete!</strong>
```
Comment (Jianghai-Quantil) [inline docs/portal/image-registry/managing-repositories-under-a-project.md:12]: **Delete** is not properly rendered on page preview. I notice that you sometimes use <strong> label pair, and sometimes use double asteriks. Why?

File: docs/portal/image-registry/viewing-logs.md [hunk 3]
```diff
@@ -2,12 +2,10 @@
...
 The Access Logs page lists the log entries associated with the selected project. The most current entries appear at the top of the list. Filters at the top of the page allow you to filter entries by registry user, repository, operation, tag, and time range.<br>
 **Note:** The more filters you use, the narrower the results. For example, if you enter criteria for User, Repository, and Operation, the log records must match all three sets of criteria to appear in the log list. Records that match one or two criteria are not shown.</br>
-   ![null](</docs/resources/images/registry/log-project.png>)
+<p align=center><img src="/docs/resources/images/registry/log-project.png" width="600"></p>
 3. Buttons and fields at the bottom right allow you to view other pages and increase the number of entries shown on a page.
 4. When you finish viewing, click the **Cancel** button at the bottom right.
```
Comment (Jianghai-Quantil) [inline docs/portal/image-registry/viewing-logs.md:11]: On page preview, this step 4 is displayed immediately after step 3 without a line break in-between. 
Besides, \*\*Cancel\*\* is not rendered properly as a bold text.

File: docs/portal/dashboard.md (hunk 1 above)
Comment (marcfishelquantil) [reply docs/portal/dashboard.md:26]: Fixed.

File: docs/portal/image-registry/managing-repositories-under-a-project.md (hunk 2 above)
Comment (marcfishelquantil) [reply docs/portal/image-registry/managing-repositories-under-a-project.md:12]: Some times, using double asterisks following a GUI snap shot prevents the double asterisks from bolding text, as you saw in this case. In these cases, I use the <strong></strong> elements. Sorry to have missed this one.

File: docs/portal/image-registry/viewing-logs.md (hunk 3 above)
Comment (marcfishelquantil) [reply docs/portal/image-registry/viewing-logs.md:11]: Fixed.

**Pull request merged**